terraform/modules/tf_stream_alert_athena/main.tf

```diff
@@ -22,6 +22,32 @@ resource "aws_lambda_function" "athena_partition_refresh" {
   }
 }
 
+// S3 Bucket: Athena Query Results and Metastore Bucket
+resource "aws_s3_bucket" "athena_results_bucket" {
+  bucket        = "${var.results_bucket}"
+  acl           = "private"
+  force_destroy = false
+
+  tags {
+    Name = "${var.results_bucket}"
+  }
+
+  versioning {
+    enabled = true
+  }
+
+  logging {
+    target_bucket = "${var.s3_logging_bucket}"
+    target_prefix = "${var.results_bucket}/"
+  }
+}
+
+// Athena Database: streamalert
+resource "aws_athena_database" "streamalert" {
```

Comment:
I'm not sure but I'm assuming they would get an error and have to choose a different name (part of the reason we're allowing users to specify their own DB names). But also, we discussed early on that these big refactors are going to be `breaking` with no backwards compatibility...?

Reply:
I'd test it and understand what will happen, because any existing Athena+SA user could potentially have issues.  There is a pretty easy Terraform workaround, and it's to import the resource.  You can proactively build this functionality into the CLI as well

Reply:
do we do any terraform importing currently?

Reply:
Not automatically, but it should be pretty trivial to discover if the `streamalert` database exists and then import it in so it's manageable

Reply:
I'm feeling like this may be a rabbit hole but will flag it to investigate